events/elections/2020/README.md

```diff
@@ -110,8 +125,8 @@ enforced as spelled out in the [Steering Committee Election Charter].
 You will be ranking your choices of the candidates with an option for
 "no opinion". In the event of a tie, a coin will be flipped.
 
-The election will open for voting the morning September 28, 2019 via email and
-end two weeks after on October 2, 2019 at 00:00am UTC. You will receive an email
+The election will open for voting on September 14th and 15th via email and
+end three weeks after on October 6, 2020 at 00:00am UTC. You will receive an email
 to the address on file at the start of the election from "Kubernetes (CIVS Poll
 Supervisor) `<civs@cs.cornell.edu>`, please whitelist if necessary. Detailed
```

Comment:
can we switch to "please add to your allow list" instead of using whitelist?

Reply:
ooooh, didn't see that!  pushing a change